assets/pages/digital-subscription-checkout/components/thankYouContent.jsx

```diff
@@ -0,0 +1,106 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import {
+  getIosAppUrl,
+  androidAppUrl,
+  getDailyEditionUrl,
+} from 'helpers/externalLinks';
+
+import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import { classNameWithModifiers } from 'helpers/utilities';
+
+import LeftMarginSection from 'components/leftMarginSection/leftMarginSection';
+import CtaLink from 'components/ctaLink/ctaLink';
+
+
+// ----- Types ----- //
+
+type PropTypes = {
+  countryGroupId: CountryGroupId;
+};
+
+
+// ----- Component ----- //
+
+function ThankYouContent(props: PropTypes) {
+
+  return (
+    <div className="thank-you-content">
+      <LeftMarginSection>
+        <p className={classNameWithModifiers('thank-you-content__copy', ['free-trial'])}>
+          You can use all the features free for the next 14 days,
+          and then your first monthly payment will be taken.
+        </p>
+      </LeftMarginSection>
+      <AppsSection countryGroupId={props.countryGroupId} />
+      <EmailsSection />
+    </div>
+  );
+
+}
+
+
+// ----- Auxiliary Components ----- //
+
+function AppsSection(props: { countryGroupId: CountryGroupId }) {
+
+  return (
+    <LeftMarginSection modifierClasses={['thank-you-apps']}>
+      <h2 className="thank-you-content__heading">
+        Can&#39;t wait to get started?
+      </h2>
+      <p className="thank-you-content__copy">
+        Just download the apps and login with the details you used to sign in.
+      </p>
+      <CtaLink
+        text="Download from the App Store"
+        accessibilityHint="Click to download the app on the Apple App Store"
```

Comment:
While we're changing things here, can we say '... _from_ the <relevant name> App Store'. Not in italics, obvs.